### ISSUE TITLE
Lost control of player during Emrakul, the Promised End control turn

### ISSUE THREAD

**Issue by dosfod** [magefree/mage#8148]
Hi, 

this has happened to me twice today, during games against AI. I cast [[Emrakul, the Promised End]] and gain control of another player during their next turn. It seems to work, I can see their hand and all... but I can't control any of the player actions. Attached screenshot with the game log:

![image](https://user-images.githubusercontent.com/29545637/130474404-bb4c293d-5ff7-4baa-a56f-12bcb4b0a171.png)

They get their "extra turn" after the one I controlled and all.

Regards

**Comment by jeffwadsworth** [comment]
Did it work correctly for you in the past?

**Comment by dosfod** [comment]
A while ago, yeah. Can’t say when exactly though

**Comment by JayDi85** [comment]
>> It seems to work, I can see their hand

You can see/request the hand from a computer player at any time.

**Comment by dosfod** [comment]
Yeah but the “Switch Hands” button is not always visible, only when there is an effect that let’s you… like Emrakul’s ability here. That’s what I meant

**Comment by siyiye** [comment]
in Latest update: 2024-09-09 11:34
this bug is back!

**Comment by JayDi85** [comment]
Fixed in #12878
